src/fu-util.c

```diff
@@ -2617,6 +2578,46 @@ fu_util_upload_security (FuUtilPrivate *priv, gchar **values, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_util_security (FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(GPtrArray) attrs = NULL;
+	g_autofree gchar *str = NULL;
+
+	/* not ready yet */
+	if ((priv->flags & FWUPD_INSTALL_FLAG_FORCE) == 0) {
+		g_set_error_literal (error,
+				     FWUPD_ERROR,
+				     FWUPD_ERROR_NOT_SUPPORTED,
+				     "The HSI specification is not yet complete. "
+				     "To ignore this warning, use --force");
+		return FALSE;
+	}
+
+	/* TRANSLATORS: this is a string like 'HSI:2-U' */
+	g_print ("%s \033[1m%s\033[0m\n", _("Host Security ID:"),
+		 fwupd_client_get_host_security_id (priv->client));
+
+	/* print the "why" */
+	attrs = fwupd_client_get_host_security_attrs (priv->client,
+						      priv->cancellable,
+						      error);
+	if (attrs == NULL)
+		return FALSE;
+	str = fu_util_security_attrs_to_string (attrs);
+	g_print ("%s\n", str);
+
+	/* ask if we can upload them to the LVFS */
+	if (!priv->assume_yes) {
+		g_print ("\n%s [y|N]: ",
+			/* TRANSLATORS: ask the user to share */
+			 _("Can we upload these anonymous results to the LVFS?"));
+		if (!fu_util_prompt_for_boolean (FALSE))
+			return TRUE;
```

Comment:
What do you think about the idea for having a config file option to automatically do this and the client can change the daemon setting like we do for reports?  I think it only makes sense if security attributes are cached and only upload for changes though.

Reply:
But part of my thought is that if that's done a GUI client could potentially be sourcing and using the same option like we have configured the update reports to be able to be done in a GUI tool.

Reply:
Aha, we have `ModifyConfig()` - sure, will give it a try.

Reply:
And also `fu_util_maybe_enable_automatic` which can hopefully be re-used

Reply:
Actually, maybe we want to tie this to a remote? it means I don't have to hardcode the URL for the LVFS... and if there's no enabled remotes then we don't offer to upload it at all. Then we use te more conventional `ModifyRemote()`....

Reply:
And even further down the rabbithole; do we want

    @@ -5,6 +5,8 @@ Enabled=true
     Title=Linux Vendor Firmware Service
     MetadataURI=https://cdn.fwupd.org/downloads/firmware.xml.gz
     ReportURI=https://fwupd.org/lvfs/firmware/report
    +SecurityURI=https://fwupd.org/lvfs/hsireports/upload
     OrderBefore=fwupd
     AutomaticReports=false
    +AutomaticHsiReports=false
     ApprovalRequired=false

Or are security reports the "same" as firmware reports?

Reply:
Yes I think that's perfect.  Then someone can opt into upload reports and security reports separately.  My vision for this is that something like gnome-software will send upload reports and something like gnome control center will send security reports.  Those "may" be separate opt in events sending separate data, so it's good to treat them as such.

Reply:
`fwupd_remote_get_security_uri` or `fwupd_remote_get_hsi_uri` or `fwupd_remote_get_security_report_uri`

Reply:
`fwupd_remote_get_security_report_uri` or `fwupd_remote_get_security_uri` to me.  Remember the command is currently `security`, not `hsi` so the conf options should match that too.

Reply:
I've added this, although I opted to ask multiple questions rather than use `fu_util_maybe_enable_automatic()` to improve the flow a bit.